status/reporters.go

```diff
@@ -226,6 +227,44 @@ func (r *GitHubReporter) createCheckRunAdapter(k8sClient client.Client, ctx cont
 	}, nil
 }
 
+func (r *GitHubReporter) createCheckRunAdapterForSnapshot(snapshot *applicationapiv1alpha1.Snapshot, integrationTestStatusDetail gitops.IntegrationTestStatusDetail, owner, repo, sha string) (*github.CheckRunAdapter, error) {
+	var title, conclusion, summary string
+	snapshotName := snapshot.Name
+	scenarioName := integrationTestStatusDetail.ScenarioName
+
+	switch integrationTestStatusDetail.Status {
+	case gitops.IntegrationTestStatusEnvironmentProvisionError:
+		title = "snapshot " + snapshotName + " experienced error when provisioning environment for integrationTestScenario " + scenarioName
+		conclusion = "errorOccured"
```

Comment:
These conclustions should be defined as constants

Reply:
Yes, we have existing constant to use.

Reply:
fixed.